views/api.jade

```diff
@@ -0,0 +1,75 @@
+extend base_layout
+  
+block head
+  link(rel='stylesheet', href="/public/css/carry.css")
+
+block page_content
+  .container
+      header.text-center
+        h1 The OpenDota API
+        a(href="//odota.github.io/docs")
+          button#thething.btn.btn-warning View API Docs
+      hr
+      .row
+        #alert.alert.alert-danger(role="alert", style="display:none;")
+        .col-md-12
+          .text-center
+            h4 Want to build your own service on top of our API?
+            p It's cheap to get started & keeps OpenDota running.
+          table.table
+            tr
+              th First 25000 Calls/month
+              th Any Calls >25000/month
```

Comment:
should be 25000+ calls/month?

Reply:
Well, even if they use 26000 calls, they'd only be paying for 1000 calls.